src/Authentication/Passwords/PwnedValidator.php

```diff
@@ -35,7 +35,8 @@ public function check(string $password, ?User $user = null): Result
     {
         $hashedPword = strtoupper(sha1($password));
         $rangeHash   = substr($hashedPword, 0, 5);
-        $searchHash  = substr($hashedPword, 5);
+        /** @var string $searchHash */
+        $searchHash = substr($hashedPword, 5);
```

Comment:
on php 7.4, `substr()` can return `false`, that's why it uses `@var` here so phpstan now that it always return string